### PR TITLE
netlink: retrieve VNI and VLAN id

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-scale-tests

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

File: topology/probes/netlink.go [hunk 1]
```diff
@@ -57,6 +57,8 @@ type neighbor struct {
 	MAC       string
 	IP        string   `json:"IP,omitempty"`
 	State     []string `json:"State,omitempty"`
+	Vlan      int      `json:"Vlan,omitempty"`
+	VNI       int      `json:"VNI,omitempty"`
```
Comment (safchain) [inline topology/probes/netlink.go]: I do not see where this field is used

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Pull request merged**